src/schema/annotation.yaml

```diff
@@ -72,6 +72,7 @@ classes:
     todos:
       - decide if this should be used for product naming (Duncan, 2021-04-02)
       - Retaining this even after removing Reaction. See todos on the Pathway and OrthologyGroup subclasses.
+    deprecated: "not used or planned to be used. 2024-07-09 https://github.com/microbiomedata/nmdc-schema/issues/1881"
```

Comment:
thanks for including the issue URLs

Why do you say "not planned to be used"? I can imagine us using a `Pathway` class in the future.

----

I also thought we were going to deprecate `Reaction`. We certainly couldn't remove `FunctionalAnnotationTerm` without either removing or re-rooting `Reaction`

Maybe the `todos` about `Reaction`s are confusing?

Reply:
maybe I saw this in nmdc-schema! may not be a problem!

Reply:
Updated the description to just say "not used", in case we model something similar in the future (from discussion metadata meeting 10 July 2024)
And I think we found that `Reaction` no longer exists in Berkeley schema anyway so I think we're good! thanks Mark!